admiral/envs/components/agent.py

```diff
@@ -98,13 +134,16 @@ def configured(self):
         """
         return super().configured and self.min_health is not None and self.max_health is not None and self.is_alive is not None
 
+class LifeObservingAgent(ObservingAgent): pass
+class HealthObservingAgent(ObservingAgent): pass
 
 
 # ----------------- #
 # --- Observing --- #
 # ----------------- #
 
-class AgentObservingAgent(Agent):
+# TODO: move this to a more specific location
```

Comment:
Resolve this TODO, possibly make a ticket for renaming or rethinking the distance-restriction.

Reply:
Added as #51